deploycron/__init__.py

```diff
@@ -42,7 +38,50 @@ def deploycron(filename="", content="", override=False):
     if installed_content:
         installed_content += "\n"
     # install back
-    retcode, err, out = _runcmd("crontab", installed_content)
+    _install_content(installed_content)
+
+def undeploycron_between(start_line, stop_line):
+    """uninstall crontab parts between two lines (included).
+    If the start_line or the stop_line is not found into the installed crontab,
+    it won't be modified.
+    start_line - start line to delimit the crontab block to remove
+    stop_line - stop line to delimit the crontab block to remove
+    """
+    lines_installed = [x.strip() for x in _get_installed_content().splitlines()]
+    start_line = start_line.strip()
+    stop_line = stop_line.strip()
+    if start_line not in lines_installed:
+        return False
+    if stop_line not in lines_installed:
+        return False
+    between = False
```

Comment:
consider the following bug cases？
- there are duplicated start_line/end_line in the installed crontabs
- the position of start_line is behind the end_line